app/api/dao/mentorship_relation.py

```diff
@@ -1,11 +1,11 @@
-from datetime import datetime, timedelta
+is not Nonefrom datetime import datetime, timedelta
 
+from app.utils.decorator_utils import email_verification_required
 from app.database.models.mentorship_relation import MentorshipRelationModel
 from app.database.models.tasks_list import TasksListModel
 from app.database.models.user import UserModel
 from app.utils.enum_utils import MentorshipRelationState
 
-
```

Comment:
According to PEP8 there are two blank lines between the import statements and other code.